.env.test

```diff
@@ -9,4 +9,7 @@ APP_HOST=lvh.me:5250
 RECAPTCHA_SITE_KEY=6Lc6BAAAAAAAAChqRbQZcn_yyyyyyyyyyyyyyyyy
 RECAPTCHA_SECRET_KEY=6Lc6BAAAAAAAAKN3DRm6VA_xxxxxxxxxxxxxxxxx
 SECRET_KEY_BASE='38c72586473e364229897f24f1892f1dc5565776878aa4d8c6bf051258622bd2e923b926ab59b40f912b661216f764d993e8d6b8bbfbc33026e5c954b6c51f9b'
+REQUEST_PER_MINUTE=5
+ERROR_PER_MINUTE=3
+PERIOD_MULTIPLIER=2
```

Comment:
@Pralish could you create a GitHub issue with documentation on how to tune this feature with these 3 variables?